### PR TITLE
deepcopy to Link.copy

### PR DESCRIPTION
**Pull request by Crissman**
Fixing Issue #1953

### CODE REVIEW

File: docs/source/tutorial/gpu.rst [hunk 1]
```diff
@@ -360,7 +360,7 @@ In order to make a copy, we can use :meth:`Link.copy` method.
 .. testcode::
 
    import copy
```
Comment (toslunar) [inline docs/source/tutorial/gpu.rst]: It seems that the module `copy` is no longer used

**Comment by toslunar** [comment]
It seems to me that this PR needs #4075 to be merged.

**Comment by delta2323** [comment]
It seems so. The modified code does not works properly unless #4075 is merged.

**Comment by delta2323** [comment]
jenkins, test this please.

**Comment by Crissman** [comment]
Jenkins, test this please

**Comment by takagi** [comment]
Now #4075 is merged.

File: docs/source/tutorial/gpu.rst [hunk 2]
```diff
@@ -359,8 +359,7 @@ In order to make a copy, we can use :meth:`Link.copy` method.
 
 .. testcode::
 
-   import copy
-   model_1 = copy.deepcopy(model_0)
+   model_1 = L.copy(model_0)
```
Comment (toslunar) [inline docs/source/tutorial/gpu.rst]: `model_1 = model_0.copy()`

(`L` is the module `chainer.links`, not the class `chainer.Link`.)

**Comment by kmaehashi** [comment]
LGTM!

**Pull request merged**